features/additional-campaign-pixel-params.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_meta": {
+        "description": "This feature controls the addition of the additional pixel params related to privacy pro campaigns.",
```

Comment:
Should this be a subfeature of privacy pro then?

Reply:
@jonathanKingston Hello! This is actually partly related to RMF and Ppro. The feature has been built to add additional params to any origins defined in this feature. The initial campaign we are using this is for ppro and appending the params to subscription pixels and RMF pixels.

I think the feature is in the intersection of RMF and PPro that’s why we put it outside (cc @cmonfortep) Also, the code for this has been merged months ago and we are just enabling the feature on a campaign starting on Monday.

We can improve / remove this once the campaign completes.

Reply:
I’ll explore that if we decide to keep this feature after the campaign. Thank you!